app/src/main/AndroidManifest.xml

```diff
@@ -36,6 +36,10 @@
     <activity android:name=".ChecklistItemActivity">
     </activity>
     <activity android:name=".AboutActivity"/>
+    <provider
+        android:authorities="com.g11x.checklistapp.provider"
+        android:name=".LocalRepository"
```

Comment:
These names are like tables in a database, right? If so, how about something specific to the items in the table, like `.ChecklistItems`?